robot/rospackages/src/mcu_control/scripts/ArmNode.py

```diff
@@ -197,7 +104,7 @@ def stripFeedback(data):
     rospy.init_node(node_name, anonymous=False) # only allow one node of this type
     rospy.loginfo('Initialized "'+node_name+'" node for pub/sub/service functionality')
 
-    init_serial()
+    init_serial(115200, mcuName)
```

Comment:
so for these magic variables that span accross different files they should be declared in a dictionary in `serial_util.py` so that they can be imported like that.

[Here's](https://github.com/PeterGhimself/BranchListUpdater) an example, look at `slackconfig.py` where the dict is defined and `branch_list_updater.py` where it is used

Reply:
Fixed